### PR TITLE
fix: Email suggestion styles

### PR DESCRIPTION
**Pull request by mubbsharanwar**
- email suggestion style fix according to figma style.

VAN-601

### CODE REVIEW

File: src/redesign/register/RegistrationPage.jsx [hunk 1]
```diff
@@ -400,7 +400,7 @@ class RegistrationPage extends React.Component {
               onClick={e => { this.handleSuggestionClick(e, this.state.emailErrorSuggestion); }}
             >
               {this.state.emailErrorSuggestion}
-            </Alert.Link>?
+            </Alert.Link>?<Icon src={Close} className="alert-close" onClick={this.handleOnClose} tabIndex="0" />
```
Comment (waheedahmed) [inline src/redesign/register/RegistrationPage.jsx:403]: I guess it should be an `IconButton` otherwise, nothing will happen on hover.

**Review by waheedahmed — CHANGES_REQUESTED** [review]
The link color is still not according to Figma, also update the Paragon version in this same PR to test all these changes.
Screenshot 2021-06-28 at 12 31 38 PM

File: src/redesign/register/RegistrationPage.jsx (hunk 1 above)
Comment (mubbsharanwar) [reply src/redesign/register/RegistrationPage.jsx:404]: Using `IconButton` background color will display on hover. We are using `alert-close` custom class in which `cursor: pointer`.

File: src/redesign/_style.scss [hunk 2]
```diff
@@ -632,24 +632,40 @@ select.form-control {
 }
 
 .alert .alert-icon {
```
Comment (waheedahmed) [inline src/redesign/_style.scss:634]: This shouldn't apply to all alert icons and instead, only apply to the email alert icon. For example, check the error icon in the following screenshot which is not aligned with the header text of the alert due to this:
Screenshot 2021-06-29 at 2 31 06 PM

File: src/redesign/_style.scss [hunk 3]
```diff
@@ -632,24 +632,40 @@ select.form-control {
...
+  margin-top: 4px;
 }
-.alert-dismissible .close {
-  top: 4px;
-  font-weight: 500;
 
-  :hover {
-      font-weight: 700;
-    }
+.alert-close {
```
Comment (waheedahmed) [inline src/redesign/_style.scss:638]: Same here, make it email alert specific.

File: src/redesign/_style.scss [hunk 4]
```diff
@@ -631,25 +631,45 @@ select.form-control {
...
-  margin-top: 3px;
-}
-.alert-dismissible .close {
-  top: 4px;
-  font-weight: 500;
+.email-error-alert {
+  padding: 0.5rem 1rem;
+
+  .alert-icon {
+    margin-top: 3px;
```
Comment (waheedahmed) [inline src/redesign/_style.scss:638]:
```suggestion
    margin-top: 4px;
```

**Review by waheedahmed — APPROVED** [review]
One small nit otherwise 👍

**Pull request merged**